### ISSUE TITLE
[BUG] disk resize for microshift 4.13.0 bundle is not working

### ISSUE THREAD

**Issue by anjannath** [crc-org/crc#3705]
#### `$ crc start --log-level debug` o/p:
```
INFO CRC VM is running
DEBU Running SSH command: cat /home/core/.ssh/authorized_keys
DEBU SSH command results: err: <nil>, output: ecdsa-sha2-nistp521 AAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAAAIbmlzdHA1MjEAAACFBAHeav7GX53Pca2b9/tRl7KQwlZQLcMrzZCsUnI3ZLNPdWn5SkKZDwTl3uoeWwunYbyu3doCq7QLeSXPtH03iXtiNQFxN60KGH/tioE0UdA/+EA5VMakD3ODsK/gbQJ+rPtiXDfqh81gQD4uWELTtpruCGjS7f546cE4u6SfSNsiDNiaXw== core
INFO Updating authorized keys...
DEBU Creating /home/core/.ssh/authorized_keys with permissions 0644 in the CRC VM
DEBU Running SSH command: <hidden>
DEBU SSH command succeeded
DEBU Running SSH command: rm /home/core/.ssh/authorized_keys.d/ignition
DEBU SSH command results: err: Process exited with status 1, output:
DEBU Using root access: Get PV disk path
DEBU Running SSH command: sudo /usr/sbin/pvs -S lv_name=root -o pv_name --noheadings
DEBU SSH command results: err: <nil>, output:   /dev/vda3
DEBU Using root access: Growing /dev/vda3 partition
DEBU Running SSH command: sudo /usr/bin/growpart /dev/vda 3
DEBU SSH command results: err: <nil>, output: CHANGED: partition=3 start=2050048 old: size=62961664 end=65011711 new: size=81835999 end=83886046
DEBU Using root access: Resizing the physical volume(PV)
DEBU Running SSH command: sudo /usr/sbin/pvresize /dev/vda3
DEBU SSH command results: err: <nil>, output:   Physical volume "/dev/vda3" changed
  1 physical volume(s) resized or updated / 0 physical volume(s) not resized
DEBU Using root access: Get the volume group size
DEBU Running SSH command: sudo /usr/sbin/vgs --noheadings --nosuffix --units b -o vg_size
DEBU SSH command results: err: <nil>, output:   41896902656
DEBU Using root access: Get the size of root logical volume
DEBU Running SSH command: sudo /usr/sbin/lvs -S lv_name=root --noheadings --nosuffix --units b -o lv_size
DEBU SSH command results: err: <nil>, output:   16106127360
INFO Extending and resizing '/dev/rhel/root' logical volume
DEBU Using root access: Extending and resizing the logical volume(LV)
DEBU Running SSH command: sudo /usr/sbin/lvextend -r -L +9663676416b /dev/rhel/root
DEBU SSH command results: err: Process exited with status 5, output:
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU Making call to close connection to plugin binary
DEBU (crc) DBG | time="2023-06-12T13:05:42+05:30" level=debug msg="Closing plugin on server side"
Error updating filesystem size: ssh command error:
command : sudo /usr/sbin/lvextend -r -L +9663676416b /dev/rhel/root
err     : Process exited with status 5
```

----

SSH into the VM and running the `lvextend` command gives the following error:

```
[core@api ~]$ sudo /usr/sbin/lvextend -r -L +9663676416b /dev/rhel/root
  LV /dev/mapper/rhel-root mounted at /sysroot may have been renamed (from /dev/mapper/rhel-root).
  LV /dev/mapper/rhel-root mounted at / may have been renamed (from /dev/mapper/rhel-root).
  LV /dev/mapper/rhel-root mounted at /etc may have been renamed (from /dev/mapper/rhel-root).
  LV /dev/mapper/rhel-root mounted at /usr may have been renamed (from /dev/mapper/rhel-root).
  LV /dev/mapper/rhel-root mounted at /sysroot/ostree/deploy/rhel/var may have been renamed (from /dev/mapper/rhel-root).
  LV /dev/mapper/rhel-root mounted at /var may have been renamed (from /dev/mapper/rhel-root).
  File system resizing not supported: fs utilities do not support renamed devices.
```

**Comment by praveenkumar** [comment]
- https://access.redhat.com/solutions/7016718
- https://bugzilla.redhat.com/show_bug.cgi?id=2211856

Looks like there is issue with `lvextend`, which doesnt' work with extend + resize and those operation need to do separately as mentioned in solution/BZ.

We should revert #3706 once it fixed as part of RHEL.

**Comment by praveenkumar** [comment]
It is fixed and #3706 is merged.
